docs/openapi.md

```diff
@@ -41,6 +41,13 @@ paths:
                   $ref: '#/components/schemas/v1.<YOUR_PAYLOAD>' # a reference to the registered type 
 ```
 
+Operation naming convention:
+
+* GET /resource - getResourceList
+* POST /resource - addResource
```

Comment:
Just a NIT, I do not care much, just curious: I see others using `create` instead of `add` is there a reason to use add ?

Reply:
No other reason then that I saw that in an OpenAPI example here: https://swagger.io/docs/specification/paths-and-operations/ and we already used add too. I can rename if you want, this is basically an internal identifier which is used by client generators at most.

Reply:
I renamed this to create*. I have no strong feelings about this.